src/forms/formHelpers.js

```diff
@@ -538,6 +539,51 @@ const MonthlyCashFlowRow = function ({ inputProps, baseValue, setClientProperty,
 };  // End <MonthlyCashFlowRow>
 
 
+class ControlledRadioYesNo extends Component {
+  constructor(props ){
+    super(props)
+    this.state = {
+    }
+  }
+
+  onChange(e,inputProps){
+    inputProps.checked = inputProps.label === 'Yes' ? true : false;
+    var obj = { ...inputProps, value: inputProps.value };
+    this.props.setClientProperty( e, obj );
+  }
```

Comment:
should be renamed to `this.props.onChange`, because you could reuse it in cases where you're not setting a client property.

Reply:
@shp54: Hmm, this would require a deeper change where this prop is passed in. May be worth it in the long run, but may be a better long term goal, not an immediate issue.